test/unit/controllers/typeController.spec.js

```diff
@@ -0,0 +1,128 @@
+/* eslint-disable no-undef,one-var-declaration-per-line,no-unused-vars,one-var */
```

Comment:
Rename to NotificationTypeController.spec.js

Reply:
done